python/ray/worker.py

```diff
@@ -1231,6 +1238,11 @@ def connect(node,
     # will use glog, which is intialized in `CoreWorker`.
     ray.state.state._initialize_global_state(
         node.redis_address, redis_password=node.redis_password)
+    if mode == SCRIPT_MODE:
+        runtime_env.upload_runtime_env_package_if_needed(job_config)
```

Comment:
we'll need another condition here to handle ray client.

Reply:
For ray client, I plan to upload the runtime to URI in the client setup, so it'll be the same as here I guess?